src/sentry/integrations/jira/notify_action.py

```diff
@@ -130,6 +130,7 @@ def get_dynamic_form_fields(self):
                     dynamic_form_fields = transform_jira_fields_to_form_fields(fields)
                     self.data["dynamic_form_fields"] = dynamic_form_fields
                     # TODO should I wipe out the rest of the data?
+                    dynamic_form_fields["rule_type"] = "jira"
```

Comment:
I added this so we can look up something specific on the front end to see if it's a ticket rule type of action. We'll need the same thing in Azure.

Reply:
@ceorourke isn't this going to create a form field option called `rule_type`? This doesn't match the usual form field schema since it's just a string value.